### ISSUE TITLE
RS Silicon not being seen by auto crafter - It wants AE-2 version Silicon

### ISSUE THREAD

**Issue by SirTwsted** [raoulvdberge/refinedstorage#1523]
**#### Issue description:**
The autocrafter is only seeing AE-2 Silicon but not recognizing that I have 2-k silicon in the system already cooked from Nether Quartz
So when I want to craft something that requires Silicon, even though I have 2-k silicon items in the system, it says it does not see it (missing items) "as it says".  And it's only seeing that I have 2-k silicon in the system but they are of AE-2 version, So it wont take them
Even though it does say "Silicon" and is the same in everything else. 

**#### What happens:**

**#### What you expected to happen:**
I would like the RS to recognize both AE-2 and RS silicons to auto craft items.

**#### Steps to reproduce:**
1. Place Silicon in RS system / Drives
2. Request RS to craft 16-k storage part
3. IT FAILS - requiring silion when system has 2-k of them  (smacks forehead in disbelief)
Looks like AE-2 and RS have their wires crossed somewhere and not playing nice with each other  :P
...

**#### Version (make sure you are on the latest version before reporting):**

- Minecraft: 1.12.2
- Forge: 14.23.0.2529
- Refined Storage: 1.5.21

**Does this issue occur on a server? [yes/no]**  -- Im playing in Private Mode / Single Player

**#### If a (crash)log is relevant for this issue, link it here:**

**[pastebin/gist/etc link here]**
This is just showing a picture saying its missing :/  when I have a bunch of them in the system 
https://imgur.com/a/SSqTW

**Comment by raoulvdberge** [comment]
Is your pattern an oredict one, so it accepts both?

**Comment by SirTwsted** [comment]
the pattern for the chip flips through both IC2 & RS
One would think that if it did that, the game is seeing that its an oredict of both.

**Comment by raoulvdberge** [comment]
A pattern doesn't "flip", what do you mean?

**Comment by SirTwsted** [comment]
For example:
You are looking at a recipe. Then you see certain parts ""flipping"" from one mod's version to another of the same item. To me thats "flipping"
Lets take a IC2 circuit, we all know how an IC2 circuit is built, 
Then it flips and shows you another mods Circuit, then It flips again and shows you another mods version of it. Then it goes back to the first one. hence the Flipping of recipe pattern.
I made sure that I selected the IC2 version of the circuit & casing when both flips the pattern recipe.
